game/settings/settings.py

```diff
@@ -407,6 +407,12 @@ class Settings:
         section=PERFORMANCE_SECTION,
         default=True,
     )
+    perf_moving_convoys: bool = boolean_option(
+        "Convoys drive the full distance between control points",
```

Comment:
The name and the description do not match. This should be named `convoys_travel_full_distance` if that's the behavior it controls.

Reply:
Renamed as suggested.